constants/xpath_consts.py

```diff
@@ -0,0 +1,8 @@
+class XpathConsts:
```

Comment:
Would be better to store locators in relations with pages. So StartPageCosts instead of XpathConsts